main.go

```diff
@@ -129,18 +172,18 @@ func main() {
 
 	daemonCommand.PersistentFlags().String(f.Service.Deployer.Installer.Configurer.File.Path, "", "Path to values file.")
 
-	daemonCommand.PersistentFlags().String(f.Service.Deployer.Installer.Configurer.Configmap.Address, "", "Address of Kubernetes API server.")
-	daemonCommand.PersistentFlags().String(f.Service.Deployer.Installer.Configurer.Configmap.CAFilePath, "", "Path to CA file for Kubernetes API server.")
-	daemonCommand.PersistentFlags().String(f.Service.Deployer.Installer.Configurer.Configmap.CrtFilePath, "", "Path to crt file for Kubernetes API server.")
-	daemonCommand.PersistentFlags().Bool(f.Service.Deployer.Installer.Configurer.Configmap.InCluster, true, "Whether draughtsman is running inside a Kubernetes cluster.")
+	daemonCommand.PersistentFlags().String(f.Service.KubernetesAddress, "", "Address of Kubernetes API server.")
+	daemonCommand.PersistentFlags().String(f.Service.KubernetesCAFilePath, "", "Path to CA file for Kubernetes API server.")
+	daemonCommand.PersistentFlags().String(f.Service.KubernetesCrtFilePath, "", "Path to crt file for Kubernetes API server.")
+	daemonCommand.PersistentFlags().Bool(f.Service.KubernetesInCluster, true, "Whether draughtsman is running inside a Kubernetes cluster.")
```

Comment:
IMO it would be nice when we align with the common structures we already have in other services. See for instance https://github.com/giantswarm/kubernetesd/blob/master/main.go#L142-L146.